TYPO3.Flow/Classes/TYPO3/Flow/Cache/CacheManager.php

```diff
@@ -358,7 +358,7 @@ protected function flushConfigurationCachesByChangedFiles(array $changedFiles)
         $objectConfigurationCache = $this->getCache('Flow_Object_Configuration');
         $caches = array(
             '/Policy\.yaml/' => array('Flow_Security_Authorization_Privilege_Method', 'Flow_Persistence_Doctrine', 'Flow_Persistence_Doctrine_Results', 'Flow_Aop_RuntimeExpressions'),
-            '/Routes([^\/]*)\.yaml/' => array('Flow_Mvc_Routing_Route', 'Flow_Mvc_Routing_Resolve'),
+            '/Routes|Settings([^\/]*)\.yaml/' => array('Flow_Mvc_Routing_Route', 'Flow_Mvc_Routing_Resolve'),
```

Comment:
"Or we make this clevererer so that it only triggers this when a setting in TYPO3.Flow.mvc.routes changed..!?" - I see not much other than that. Could be a cool feature anyway to only flush specific caches depending on specific settings.